app/assets/src/stylesheets/01-atoms/buttons.scss

```diff
@@ -54,3 +57,19 @@ $_button-text-color: $white;
     }
   }
 }
+
+.button--secondary {
```

Comment:
I know a lot of people love the `.repeated-element-name--variation` part of BEM, but I kinda feel that if you're in a situation where `.repeated-element-name.--variation` isn't good enough then there might be something else wrong?

For consistency's sake, please either make this `.--secondary` or change the way we're handling variation names in layout.

Reply:
Does Sass support?

```
button {
  &.--secondary
}
```

I'm not attached to anything about BEM. I honestly literally don't know the practical rules of this new world of UI. For instance, syntactical examples of what you want is preferable to philosophical things since I more need to know the *what* and *how* since this is all new to me.

Reply:
It does! I am fine with that syntax if it's more readable for you than `button.--secondary`. I kind of have a twitch reaction to `&` because it gets used as an excuse to over-nest a lot, but I can try to get over it.

Reply:
Yea, the problem is that bourbon defines this `$all-buttons` variable, which I suppose I could *probably* re-use instead of doing `&.`? I just need to poke.